cmd/webhook/main.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 The Casdoor Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/unrolled/secure"
+
+	"github.com/casbin/k8s-authz/internal/handler"
+)
+
+func tlsHandler(c *gin.Context) {
+	secureMiddleware := secure.New(secure.Options{
+		SSLRedirect: true,
+		SSLHost:     "localhost:8080",
```

Comment:
SSL host is a fixed value,  or only for development now?

Reply:
For development now. It will be changed to ENV when time is ripe